packages/design/src/Radii.stories.mdx

```diff
@@ -26,8 +24,8 @@ options or the "pip" in
 
 export function Example({ of: radius }) {
   const style = {
-    width: "var(--space-large)",
-    height: "var(--space-large)",
+    width: "64px",
```

Comment:
Hard-coding new values here because we're not actually tying this size to any spacing values, this is purely optical.